gpflow/inducing_variables/inducing_variables.py

```diff
@@ -69,7 +70,7 @@ class Multiscale(InducingPointsBase):
       }
     """
 
-    def __init__(self, Z, scales):
+    def __init__(self, Z: TensorType, scales: TensorType):
```

Comment:
```suggestion
    def __init__(self, Z: TensorData, scales: TensorData):
```
?

Reply:
it was because of the `.shape` access on `Z` and `scales`, but I realise I can work round that